pkg/Microsoft.ML.TensorFlow/Microsoft.ML.TensorFlow.nupkgproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
```

Comment:
@ericstj - We should also have a ".symbols" pkgproj file.  That way a symbols package gets produced and is uploaded to the symbols server for the managed assemblies in this package.  See the other folders for an example.